src/app/components/your-work/your-work.component.html

```diff
@@ -1 +1,49 @@
+<ng-template #tableTemplatePartial let-row>
+  <td class="govuk-table__cell">{{ row.case_number }}</td>
+  <td class="govuk-table__cell">{{ row.courthouse_name }}</td>
+  <td class="govuk-table__cell">{{ row.hearing_date | date: 'dd MMM yyyy' }}</td>
+  <td class="govuk-table__cell">{{ row.transcription_type }}</td>
+  <td class="govuk-table__cell">{{ row.requested_ts | date: 'dd MMM yyyy HH:mm' }}</td>
+  <td class="govuk-table__cell">
+    <span [ngClass]="row.urgency === 'Overnight' ? 'moj-badge moj-badge--red' : ''">{{ row.urgency }}</span>
+  </td>
+  <td class="govuk-table__cell">
+    <a href="#" (click)="onViewWorkRequest($event, row)" class="govuk-link">View</a>
```

Comment:
I think this can just be a link to ``/work/${row.transcription_id}``, rather than a function using router?

Reply:
Removed function and used direct link as per suggestion

Reply:
I think I've probably suggested the wrong thing here, this might be better as a `[routerLink]`, maybe @deanbullock-hmcts could confirm?

Reply:
Changed to `[routerLink]`, commit can be reverted if incorrect

Reply:
I agree Dave. The routerLink is probably cleaner / less code. Josh's approach is acceptable if we needed to do other things other than navigate